### ISSUE TITLE
Create entrypoint script for docker to run migrate script

### ISSUE THREAD

**Issue by Tirokk** [Human-Connection-Community/Human-Connection-New-20201003#220]
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-03-15T15:14:49Z" title="Friday, March 15th 2019, 4:14:49 pm +01:00">Mar 15, 2019</time>_
_Closed <time datetime="2019-03-26T18:47:03Z" title="Tuesday, March 26th 2019, 7:47:03 pm +01:00">Mar 26, 2019</time>_
---

**Is your feature request related to a problem? Please describe.**
currently, one must remember to run the script as it creates unique constraints in the database

**Describe the solution you'd like**
create a entrypoint script that waits on the server and then runs the migrate script, and maybe seeds the database
**Describe alternatives you've considered**
create custom error to remind a user to run the script like rails does when one has pending migrations, this seems like it might be more work

**Comment by Tirokk** [comment]
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-03-15T15:35:53Z" title="Friday, March 15th 2019, 4:35:53 pm +01:00">Mar 15, 2019</time>_

---


@mattwr18 don't *seed* just *migrate* (== create constraints, at the moment).

**Comment by Tirokk** [comment]
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-03-15T16:26:53Z" title="Friday, March 15th 2019, 5:26:53 pm +01:00">Mar 15, 2019</time>_

---


cool, thanks for the feedback :)

**Comment by Tirokk** [comment]
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-03-26T18:47:01Z" title="Tuesday, March 26th 2019, 7:47:01 pm +01:00">Mar 26, 2019</time>_

---


closing in favor of #206
